UNITTESTS/events/equeue/test_equeue.cpp

```diff
@@ -139,11 +139,64 @@ static void *multithread_thread(void *p)
     return 0;
 }
 
+class ecount {
+    mutable pthread_mutex_t mutex;
+    pthread_cond_t cond;
+    uint8_t count;
+public:
+    ecount() : count(0)
+    {
+        int err = pthread_mutex_init(&mutex, NULL);
+        EXPECT_EQ(0, err);
+        err = pthread_cond_init(&cond, NULL);
+        EXPECT_EQ(0, err);
+    }
+
+    ~ecount()
+    {
+        int err = pthread_mutex_destroy(&mutex);
+        EXPECT_EQ(0, err);
+        err = pthread_cond_destroy(&cond);
+        EXPECT_EQ(0, err);
+    }
+
+    void lock() const
+    {
+        int err = pthread_mutex_lock(&mutex);
+        EXPECT_EQ(0, err);
+    }
+
+    void unlock() const
+    {
+        int err = pthread_mutex_unlock(&mutex);
+        EXPECT_EQ(0, err);
+    }
+
+    void touch()
+    {
+        lock();
+        if (count < 200) {
```

Comment:
Why 200? What is the significance of this value ?

Reply:
It's from the original code.

Don't think it's particularly magic - it's using an 8-bit counter for "how many times was this called", but stops incrementing at 200.